cla_common/tests/test_call_centre_availability.py

```diff
@@ -1,14 +1,11 @@
 from datetime import datetime, time
 import unittest
+import mock
```

Comment:
Is this used?

Reply:
Updated. I will create a ticket to add flake8 to this repo so that this can be caught earlier